examples/with-linaria/next.config.js

```diff
@@ -0,0 +1,19 @@
+const withCSS = require('@zeit/next-css')
+
+module.exports = withCSS({
+  webpack (config, options) {
+    const jsRule = config.module.rules.findIndex(loader => loader.test.test('test.jsx'))
+    // Dirty dirty dirty hack to make the zero runtime stuff work
+    config.module.rules[jsRule].use = [
+      config.module.rules[jsRule].use,
+      {
+        loader: 'linaria/loader',
+        options: {
+          sourceMap: process.env.NODE_ENV !== 'production'
+        }
+      }
+    ]
+
```

Comment:
I just tried out the example and it seems to work alright using the below snippet instead of the `Dirty dirty dirty hack` (nice comment by the way 😂)

```js
config.module.rules.push({
  test: /\.js$/,
  use: [
    {
      loader: 'linaria/loader',
      options: {
        sourceMap: process.env.NODE_ENV !== 'production',
      },
    }
  ],
})
```